examples/launch_tensorboard_example.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can you put a high-level comment on this example that explains what is happening? (e.g. that tensorboard is used to log metrics and that explains how to visualize the metrics)

Also it would be nice to have it link in the FAQ (something like How can I visualize the progress of my tuning experiment with Tensor board?)